_rules/visible-label-in-accessible-name-2ee8b8.md

```diff
@@ -93,6 +93,14 @@ Not all of [visible][] label is included in [accessible name][].
 <button name="link" aria-label="the full">The full label</button>
 ```
 
+#### Failed Example 3
+
+Mathematical symbol cannot be substitute for text as [explicitly mentioned in WCAG](https://www.w3.org/WAI/WCAG21/Understanding/label-in-name#mathematical-expressions-and-formulae).
```

Comment:
Mathematical symbol cannot be substitute... > Mathematical symbols cannot be substituted...

Reply:
+1 @Brynanders "Mathematical symbols cannot be substituted by text"

Reply:
Can actually go both ways, depending on which one is the "original" and which one is the [substitute](https://en.wiktionary.org/wiki/substitute#Noun) 😄

Reply:
@Jym77 its grammatically incorrect to use the singular "symbol" and "substitute"
symbol > symbols
substitute > substituted

Reply:
Ah, yes, I missed the requested pluralisation of "symbol" 😞 Makes sense…